scheduler/scheduler-server/src/test/java/performancetests/recovery/NodeRecoveryTest.java

```diff
@@ -112,14 +116,33 @@ public void startKillStartRM() throws Exception {
         assertEquals(nodesNumber, resourceManager.getState().getAllNodes().size());
     }
 
+    @Ignore
     @Test
     public void test() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1607)